Main.pas

```diff
@@ -971,13 +971,15 @@ procedure TMainForm.FormMouseWheelDown(Sender: TObject; Shift: TShiftState;
 begin
   if GetKeyState(VK_CONTROL) >= 0  then IncRit(1)
   else if RunMode <> rmHst then SetBw(ComboBox2.ItemIndex-1);
+  Handled := true;  // set Handled to prevent being called 3 times
```

Comment:
This change has been applied in many forks over time. However, none of them had detected or fixed the problem of multiple events being handled for a single mouse wheel click/movement. By setting Handled=true, only one event is handled. This was important for smooth operation of Bandwidth control using the Cntl-key.